projects/core/src/user/facade/user.service.ts

```diff
@@ -51,161 +31,11 @@ export class UserService {
     if (this.userAccountFacade) {
```

Comment:
Could we adjust return type of this method to `Observable<User | undefined>`?

Reply:
I changed a type here, however had to update some files because of error during compilation, as `undefined` value is now possible.